log/index.js

```diff
@@ -117,19 +117,21 @@ function log (client, messages = {}) {
             showLog('action ' + bold(action.id) + ' was processed')
           }
         } else if (action.type === 'logux/undo') {
-          message =
-            'action ' +
-            bold(action.id) +
-            ' was undid because of ' +
-            bold(action.reason)
-          let details = {}
+          if (action.action.type === 'logux/subscribe') {
+            message = 'subscription to ' + bold(action.action.channel)
+          } else {
+            message = 'action ' + bold(action.action.type)
+          }
+          message += ' was undone because of ' + bold(action.reason)
+          let details = {
+            'Undone Action': action.action
+          }
+          if (Object.keys(action).length > 4) {
+            details['Undo Action'] = action
```

Comment:
```suggestion
            details['Undid By'] = action
```

Reply:
Sounds like it’s about `clientId` or `nodeId`, but it’s just undo action.

Reply:
`Undid by Action`?

Reply:
Why you don’t like “Undo Action”? It will only be visible when undo action has extra fields.

Reply:
I think the name should be super simple like “Original Undo Action” or just “Undo Action”.

Reply:
I am afraid that non-native English speakers will not understand the difference between `Undo` and `Undone`.

Maybe: `Reverted action` and `Undo action`?

Reply:
Oh, you’re right. “Reverted action” sounds cool.